logFileReplay/send_data_py3.py

```diff
@@ -67,7 +66,7 @@ def send_data(metric_data):
     post_url = config_vars['server_url'] + "/customprojectrawdata"
     response = requests.post(post_url, data=json.loads(to_send_data_json), verify=False)
     if response.status_code == 200:
-        print(str(getsizeof(to_send_data_json)) + " bytes of data are reported.")
+        print(str(len(bytearray(to_send_data_json))) + " bytes of data are reported.")
```

Comment:
change to len(bytearray(json.dumps(data), 'utf8'), otherwise got an exception:
TypeError: string argument without an endcoding